resources/views/index.blade.php

```diff
@@ -61,6 +61,7 @@
                 <img class="homepage-logo" src="../img/logo-new.png" />
                 <h1 style="font-size:8vh"> Motask</h1>
                 <h3>Place Where You Can Find Your Own Projects</h3>
+                <h4>For more information and user guide, <a href="http://bit.ly/TutorialMotask">Click Me!</a></h4>
```

Comment:
nit: prefer "click me!" with all lowercase to make it consistent with the whole sentence.

Reply:
Agreed :+1: